instrumentation/jetty-httpclient/jetty-httpclient-9.2/testing/src/main/groovy/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/AbstractJettyClient9Test.groovy

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2
+
+import io.opentelemetry.context.Context
+import io.opentelemetry.context.Scope
+import io.opentelemetry.instrumentation.test.base.HttpClientTest
+import org.eclipse.jetty.client.HttpClient
+import org.eclipse.jetty.client.api.ContentResponse
+import org.eclipse.jetty.client.api.Request
+import org.eclipse.jetty.client.api.Response
+import org.eclipse.jetty.client.api.Result
+import org.eclipse.jetty.http.HttpMethod
+import org.eclipse.jetty.util.ssl.SslContextFactory
+import org.junit.Rule
+import org.junit.jupiter.api.Assertions
+import org.junit.rules.TestName
+import spock.lang.Shared
+
+import java.util.concurrent.TimeUnit
+
+abstract class AbstractJettyClient9Test extends HttpClientTest<Request> {
+
+
+//  abstract void attachInterceptor(Request jettyRequest, Context parentContext);
+  abstract HttpClient createStandardClient();
+
+  abstract HttpClient createHttpsClient(SslContextFactory sslContextFactory);
+
+
+  @Shared
+  def client = createStandardClient()
+  @Shared
+  def httpsClient = null
+
+  @Rule
+  TestName name = new TestName()
+
+  def setupSpec() {
+
+    try {
+      client.start()
+    } catch (Throwable t) {
+      Assertions.fail("Error during jetty client start", t)
+    }
+
+    SslContextFactory tlsCtx = new SslContextFactory()
+    tlsCtx.setExcludeProtocols("TLSv1.3")
+    httpsClient = createHttpsClient(tlsCtx)
+    httpsClient.setFollowRedirects(false)
+    try {
+      httpsClient.start()
+    } catch (Throwable t) {
+      Assertions.fail("Error during jetty client https start", t)
+    }
+
+  }
+
+  @Override
+  Request buildRequest(String method, URI uri, Map<String, String> headers) {
+
+    HttpClient theClient = uri.scheme == 'https' ? httpsClient : client
+
+    Request request = theClient.newRequest(uri)
+
+    HttpMethod methodObj = HttpMethod.valueOf(method)
+    request.method(methodObj)
+    request.timeout(CONNECT_TIMEOUT_MS, TimeUnit.MILLISECONDS)
+
+    return request
+  }
+
+  @Override
+  String userAgent() {
+    if (name.methodName.startsWith('connection error')) {
+      return null
+    }
+    return "Jetty"
+  }
+
+  @Override
+  int sendRequest(Request request, String method, URI uri, Map<String, String> headers) {
+    headers.each { k, v ->
+      request.header(k, v)
+    }
+
+//    def interceptor = createInterceptor(Context.current())
+    Context parentContext = Context.current()
+    Scope scope = parentContext.makeCurrent()
+//    attachInterceptor(request, parentContext)
+    ContentResponse response = request.send()
+    scope.close()
```

Comment:
There's no need to make current context current again:
```suggestion
    ContentResponse response = request.send()
```

Reply:
Done